server.js

```diff
@@ -260,9 +271,12 @@ async function initiateCallToPSTN(account_id, from_number, to_number) {
  */
 async function endCallToPSTN(account_id, call_id) {
   // call body, see here for more details: https://dev.bandwidth.com/voice/methods/calls/postCallsCallId.html
-  var body = new BandwidthVoice.ApiModifyCallRequest({ state: "completed" });
+  var body = {
+    state: "completed",
+    redirectUrl: 'foo'
```

Comment:
I believe there is a bug in the Voice SDK. This redirectUrl parameter is required in the TS interface, but the parameter is optional in the API doc: https://dev.bandwidth.com/voice/methods/calls/postCallsCallId.html

Reply:
Voice has a new spec they've submitted to use which may resolve this.

Reply:
We're planning a release with new versions of the SDKs for our next sprint which should include this change.

Reply:
We can iterate on it later but I updated to the latest, recent voice SDK and the issue is still there